packages/mco/components/modals/app-manage-policies/app-manage-policies-modal.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { StatusBox } from '@odf/shared/generic/status-box';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { Modal, ModalVariant } from '@patternfly/react-core';
+import { PolicyListView } from './policy-list-view';
+import {
+  ManagePolicyStateType,
+  ModalActionContext,
+  ModalViewContext,
+  initialPolicyState,
+  managePolicyStateReducer,
+} from './utils/reducer';
+import { ApplicationType } from './utils/types';
+
+export const AppManagePoliciesModal: React.FC<AppManagePoliciesModalProps> = (
+  props
```

Comment:
```suggestion
{isOpen, ...
```
Destructure all props here itself.